drivers/usbc/tcpc/Kconfig.tcpc_stm32

```diff
@@ -3,12 +3,10 @@
 # Copyright 2022 The Chromium OS Authors
 # SPDX-License-Identifier: Apache-2.0
 
-DT_COMPAT_ST_STM32_UCPD := st,stm32-ucpd
-
 config USBC_TCPC_STM32
 	bool "USBC TCPC device controller driver"
 	depends on SOC_FAMILY_STM32
```

Comment:
remove the `depends on SOC_FAMILY_STM32`

Reply:
Done